addons/l10n_fr_account/data/tax_report_data.xml

```diff
@@ -1519,7 +1519,13 @@
                         <field name="name">32 - Total payable</field>
                         <field name="name@fr">32 - Total à payer</field>
                         <field name="code">box_32</field>
-                        <field name="aggregation_formula">box_28.balance + box_29.balance + box_Z5.balance</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_32_balance" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">box_28.balance + box_29.balance + box_Z5.balance</field>
+                            </record> 
+                        </field>
```

Comment:
Why this change?

It doesn't seem to be related to the purpose of this PR. Moreover, the `aggregation_formula` field creates a balance expression without a xmlid, making this change non upgradable without a dedicated upgrade script.

Reply:
This was needed because we needed to reference the tax_report_32_balance expression, but I don't think it is needed anymore, and I see Saif just removed it. Thanks for noticing!